lib/fedex/tracking_information.rb

```diff
@@ -47,11 +47,15 @@ def initialize(details = {})
       if details.has_key?(:estimated_delivery_timestamp)
         @estimated_delivery_at = Time.parse(details[:estimated_delivery_timestamp])
       end
+      
+      if details[:events].select { |event| event[:event_type] == "PU" }.first
```

Comment:
as @davidamcclain mentioned in your other PR, it would be good to have "PU" pulled out as a constant so people will understand what it means without having to search through fedex documentation on tracking codes.